### PR TITLE
Add SessionViewCommand to DeveloperGuide

### PR DESCRIPTION
**Pull request by matheril**
Fixes #383

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -194,6 +194,26 @@ The general workflow of the `session /create` command is as follows:
...
+**API reference:** [`SessionViewCommand.java`](https://github.com/AY2122S2-CS2113T-T10-1/tp/blob/master/src/main/java/seedu/splitlah/command/SessionCreateCommand.java)
+
+The sequence diagram below models the interactions between various entities in SplitLah
+when the user invokes the `session /view` command.
+<br>
+<br>
+![View Session Sequence Diagram Screenshot](https://raw.githubusercontent.com/AY2122s2-cs2113t-t10-1/tp/master/docs/images/developerguide/SessionViewCommand.drawio.png))
+<br>
+<br>
+The general workflow of the `session /create` command is as follows:
```
Comment (froststein) [inline docs/DeveloperGuide.md:206]: Should change `session /view ` not `session /create`

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -194,6 +194,26 @@ The general workflow of the `session /create` command is as follows:
 
 ### Remove a session
 ### View a session
+**API reference:** [`SessionViewCommand.java`](https://github.com/AY2122S2-CS2113T-T10-1/tp/blob/master/src/main/java/seedu/splitlah/command/SessionCreateCommand.java)
```
Comment (froststein) [inline docs/DeveloperGuide.md:197]: The reference is not the correct reference.

**Review by froststein — CHANGES_REQUESTED** [review]
Some errors on the type of command being explained.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (matheril) [reply docs/DeveloperGuide.md]: Resolved

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (matheril) [reply docs/DeveloperGuide.md]: Resolved

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -194,6 +194,27 @@ The general workflow of the `session /create` command is as follows:
...
+The sequence diagram below models the interactions between various entities in SplitLah
+when the user invokes the `session /view` command.
+<br>
+<br>
+![View Session Sequence Diagram Screenshot](https://raw.githubusercontent.com/AY2122s2-cs2113t-t10-1/tp/master/docs/images/developerguide/SessionViewCommand.drawio.png)
+<br>
+<br>
+The general workflow of the `session /view` command is as follows:
+1. The user input provided is passed to `Splitlah`.
+2. `Splitlah` then parses the input by using methods in the `Parser` class to obtain a `SessionViewCommand` object.
```
Comment (warrencxw) [inline docs/DeveloperGuide.md:208]: Consider changing `Splitlah` to `SplitLah` for consistency.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -194,6 +194,27 @@ The general workflow of the `session /create` command is as follows:
...
+3. `SessionViewCommand#run` method is then invoked to run the `session /view` command.
+4. The list of sessions are stored in a `Profile` object, hence `Manager#getProfile` is called.
+5. The `SessionViewCommand` object then runs the `Profile#getSession` method to retrieve the session represented
+   by the `sessionId` provided.
+   1. If the session with the requested sessionId does not exist, an error message is printed out with 
+      `TextUI#printlnMessage`.
```
Comment (warrencxw) [inline docs/DeveloperGuide.md:214]: Consider changing both instances of `sessionId` to session unique identifier to standardise with Profile component which uses that terminology.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -209,14 +230,14 @@ The general workflow of the `session /list` command is as follows:
 2. `Splitlah` then parses the input by using methods in the `Parser` class to obtain a `SessionListCommand` object.
```
Comment (warrencxw) [inline docs/DeveloperGuide.md:230]: Likewise here, consider changing `Splitlah` to `SplitLah` for consistency

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -209,14 +230,14 @@ The general workflow of the `session /list` command is as follows:
...
-     using `TextUi#printlnMessage`.
-  2. If the list is not empty, `SessionListCommand` will loop from the first to the second last session,
-     calling `TextUi#printlnMessage()` to print out a brief overview of each session.
-     Then, the last group is printed with a divider below it, using the method `TextUi#printlnMessageWithDivider()`.
+7. Once the list is retrieved, `SessionListCommand` class checks if the list is empty.
+8. If the list is empty, a message indicating that the list is empty is printed
+   using `TextUi#printlnMessage`.
+9. If the list is not empty, `SessionListCommand` will loop from the first to the second last session,
+   calling `TextUi#printlnMessage()` to print out a brief overview of each session.
+   Then, the last group is printed with a divider below it, using the method `TextUi#printlnMessageWithDivider()`.
```
Comment (warrencxw) [inline docs/DeveloperGuide.md:240]: Numbering used here is a little weird, maybe have it be consecutive for better consistency?

I think you have to change this entire part to reflect changes (right now SessionListCommand is updated to use `Profile#getSessionListSummaryString` instead, maybe reference #375 for ActivityListCommand which uses `Session#getActivityListSummaryString`.

**Review by warrencxw — CHANGES_REQUESTED** [review]
A few consistency issues present. Added a few suggestions that you can follow to fix.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (matheril) [reply docs/DeveloperGuide.md]: Resolved across the DG

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (matheril) [reply docs/DeveloperGuide.md]: Resolved

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (matheril) [reply docs/DeveloperGuide.md]: Resolved

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (matheril) [reply docs/DeveloperGuide.md:240]: Oops, I changed this by accident. Reverted

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Review by froststein — APPROVED** [review]
Looks good to merge

**Pull request merged**